.github/workflows/ci-pipeline.yaml

```diff
@@ -95,13 +107,63 @@ jobs:
           # Container name
           VIB_ENV_CONTAINER: ${{ steps.get-container-metadata.outputs.name  }}
           VIB_ENV_TAG: ${{ steps.get-container-metadata.outputs.tag }}
-  verify-all:
+  verification-summary:
     # Ensure all containers passed the verification
     runs-on: ubuntu-latest
-    name: Verify
-    needs: vib-verify
+    name: Verification Summary
+    needs:
+      - get-containers
+      - vib-verify
     if: ${{ always() }}
     steps:
-      - name: Verify
-        if: ${{ needs.vib-verify.result != 'success' && needs.vib-verify.result != 'skipped'}}
-        run: exit 1
+      - name: Verification Summary
+        run: |
+          if [[ "${{ needs.get-containers.result }}" != "success" ]]; then
+            echo "Verification is required."
+            echo " If you've just created this PR, don't worry about this message. Bitnami Team has to review it and make possible the verification."
+            exit 1
+          else
+            if [[ "${{ needs.get-containers.outputs.result }}" == "skip" ]]; then
+              echo "It seems these changes don't involve any container"
+            else
+              if [[ "${{ needs.vib-verify.result }}" != "success" ]]; then
+                echo "Verification is required"
+                echo "Please review previous jobs to get more information"
+                exit 1
+              else
+                echo "Well done! Everything looks good. Please wait for the Bitnami Team review."
+              fi
+            fi
+          fi
+  auto-pr-review:
+    runs-on: ubuntu-latest
+    name: Reviewal for automated PRs
+    if: |
+      always() &&
+      github.actor == 'bitnami-bot'
+    needs: verification-summary
+      - name: Automerge labeling
+        uses: andymckay/labeler@1.0.4
+        with:
+          repo-token: "${{ secrets.GITHUB_TOKEN }}"
+          add-labels: auto-merge
+      - name: Enable auto-merge feature
+        if: ${{ needs.verification-summary.result == 'success' }}
+        uses: reitermarkus/automerge@v2.1.2
+        with:
+          token: ${{ secrets.BITNAMI_BOT_TOKEN }}
+          merge-method: squash
+          pull-request: ${{ github.event.number }}
+      - name: PR Approval
+        if: ${{ needs.verification-summary.result == 'success' }}
+        uses: hmarr/auto-approve-action@v2.4.0
+        with:
+          pull-request-number: ${{ github.event.number }}
+      - name: Manual review required
+        if: ${{ needs.vib-verify.result != 'success' }}
```

Comment:
Shouldn't we use `needs.verification-summary` here as well?

Reply:
You right! Good Catch